### ISSUE TITLE
ViteOptions and ViteServerOptions Do not Support ".Configure(...)"

### ISSUE THREAD

**Issue by EmperorArthur** [Eptagone/Vite.AspNetCore#54]
[ViteOptions](https://github.com/Eptagone/Vite.AspNetCore/blob/main/src/Vite.AspNetCore/ViteOptions.cs) and [ViteServerOptions](https://github.com/Eptagone/Vite.AspNetCore/blob/main/src/Vite.AspNetCore/ViteServerOptions.cs) have all properties configured as `{ get; init; }`.

This causes the following extremely common pattern to not compile!
```C#
builder.Services.Configure<ViteOptions>(options =>
{
  options.PackageDirectory = "myClientApp";
});
```

**Comment by Eptagone** [comment]
Hi. You're right. I'll change it with the next release.

**Comment by Eptagone** [comment]
Hi, the vite options records were reconfigured as `{ get; set; }`. The changes are available with [v1.10.1](https://github.com/Eptagone/Vite.AspNetCore/releases/tag/v1.10.1)
